lib/bright_serializer/serializer.rb

```diff
@@ -35,6 +35,8 @@ def serialize(object, attributes_to_serialize)
     end
 
     def serializable_hash
+      return nil if @object.nil?
```

Comment:
I think it would be a better place to make that change in the [serialize method](https://github.com/petalmd/bright_serializer/blob/aecf9072369a92672959f9d9387bc7201c0176e7/lib/bright_serializer/serializer.rb#L29-L29)